File: transektcount/src/main/java/com/wmstein/transektcount/database/Head.java
package com.wmstein.transektcount.database;

/*
 * Created by wmstein on 31.03.2016.
 */
public class Head
{
    public int id;
    public String transect_no;
    public String inspector_name;

    //setting H_ID *******************
    public void setH_ID (int H_id)
    {
        this.id = H_id;
    }

    //setting H_TRANSECT_NO
    public void setH_TRANSECT_NO(String H_transect_no)
    {
        this.transect_no = H_transect_no;
    }

    //setting H_INSPECTOR_NAME
    public void setH_INSPECTOR_NAME(String H_inspector_name)
    {
        this.inspector_name = H_inspector_name;
    }
    
}

File: transektcount/src/main/java/com/wmstein/transektcount/database/Head.kt
package com.wmstein.transektcount.database

/***********************************
 * Definitions for table Head
 *
 * Created by wmstein on 31.03.2016,
 * last edited in Java on 2022-03-23,
 * converted to Kotlin on 2023-06-26,
 * last edited on 2026-03-17
 */
class Head {
    @JvmField
    var id = 0
    @JvmField
    var transect_no: String? = null
    @JvmField
    var inspector_name: String? = null

    @JvmField
    var data_language: String? = null
}